apps/halide/blur/main.cpp

```diff
@@ -96,40 +96,23 @@ bool write_png_file(
 }
 
 typedef void (*blurtype)(
-    void *ctxt, int_fast32_t n, int_fast32_t m, uint8_t *g, const uint8_t *inp);
+    void *ctxt, int_fast32_t n, int_fast32_t m, uint16_t *g, const uint16_t *inp);
 
 int exec_parrot(blurtype func, std::string output_name, int width, int height,
```

Comment:
Should this function be renamed to `exec_func` or `exec_blur`?

Should all the buffers be renamed from `parrot*` to `inp*/img*`